docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php

```diff
@@ -240,4 +187,46 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->save();
   }
 
+  /**
+   * Gets an array of nodes, based on bundle type.
+   *
+   * @param string $bundleType
+   *   The type of bundle.
+   *
+   * @return array
+   *   Array of nodes.
+   */
+  protected function getArrayBasedOnBundleType(string $bundleType) {
```

Comment:
For a method we want the name to be as descriptive as possible about what it does, more so than the shape of the thing it returns.  The @return covers the fact that it is in array.  Can we go with a name that tells what is being returned?

Something like 'getNidsFromEntityBundleQuery('

Reply:
Thank you for this guidance:
> For a method we want the name to be as descriptive as possible about what it does

That's been updated, too.